PrestaShop/PrestaShop
# Brand address create/edit actions migration

| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | Migrate Catalog > Brands & Suppliers > Address create/edit actions
| Type?         | refacto
| Category?     | BO
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | Fixes #10544
| How to test?  | See below

## How to test

Edit and create actions should work as in legacy page except following changes:
- :warning: On create/edit action enabled selecting and reseting brand
- In form field 'others' fixed illegal characters hint  according to its validation rules.

Testing forms can be accessed from listing page at `/admin-dev/index.php/sell/catalog/brands/`, follow the click on Add or Edit button.
Or directly via links
- `/admin-dev/index.php/sell/catalog/brands/addresses/new`
- `/admin-dev/index.php/sell/catalog/brands/addresses/{addressId}/edit`


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/prestashop/prestashop/12795)
